src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1012,7 +1021,11 @@ private function getProcess($commandline, $cwd = null, array $env = null, $stdin
             }
         }
 
-        return $process;
+        if (self::$process) {
+            self::$process->stop(0);
+        }
```

Comment:
Why don't we put this into `setUp()` then? Imho that would make the class more readable given that we have the same in `tearDown()`.